csrc/kernel_ir.h

```diff
@@ -1126,5 +1127,82 @@ class TORCH_CUDA_CU_API GetRNGSeedAndOffsetFromHost : public Expr {
       const std::vector<PolymorphicValue>& inputs) const override;
 };
 
+// Expr for driver API cuTensorMapEncodeTiled
+class TORCH_CUDA_CU_API EncodeTensorMapTiled : public Expr {
```

Comment:
Don't we need to update `Expr::dispatch` and `Expr::constDispatch`?

I tried some of the tests and saw some errors due to missing dispatch handling with `NVFUSE_DISABLE=index_hoist`.

Reply:
dispatch added